src/views/Home.vue

```diff
@@ -33,6 +33,7 @@ console.log(tournaments)
 	<section class="snap-start" style="height: calc(100vh - 6rem)">
 		<div class="title text-white my-2">Cosplay</div>
 	</section>
+	<Partners />
```

Comment:
because of the home page scrolling, Partner section can't be reach just by mouse or key scrolling. You need to put it in a section with class="snap-start".